README-FIPS.md

```diff
@@ -71,11 +71,11 @@ the installation by doing the following two things:
 
 - Runs the FIPS module self tests
 - Generates the so-called FIPS module configuration file containing information
-  about the module such as the module checksum (and for OpenSSL 3.0 the
+  about the module such as the module checksum (and for OpenSSL 3.1.2 the
   self test status).
 
 The FIPS module must have the self tests run, and the FIPS module config file
-output generated on every machine that it is to be used on. For OpenSSL 3.0,
+output generated on every machine that it is to be used on. For OpenSSL 3.1.2
```

Comment:
This paragraph was intended to be no longer true with 3.1.2. Unfortunately the security policy still contains a leftover sentence that would need to be removed to make it fully correct.

Reply:
The security policy should be updated when the FIPS provider is submitted for a 3(A|B) sub update.

There are [three CVEs that impact the FIPS boundary](https://openssl-library.org/news/fips-cve/index.html) that are not addressed in 3.1.2, this will require a 3A submission to update to 3.1.6.